db/migrate/20160114011126_create_indices.rb

```diff
@@ -0,0 +1,13 @@
+class CreateIndices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :indices do |t|
+      t.datetime :finalized_at
+      t.integer :category_id
```

Comment:
This field really should be created using the references method, that way it will create the foreign key relation in the database and ensure the database stays consistent.